examples/dashboard/dashboard.py

```diff
@@ -28,7 +31,6 @@
 from webargs import fields
```

Comment:
We do need to make it easy for people to run the example, though.  After getting an error message about it, I briefly looked for webargs on conda, tried some bogus versions from non-main channels that didn't work, and eventually pip-installed webargs (which worked).  Most people probably aren't that dedicated.